near/src/runtime.rs

```diff
@@ -291,8 +291,11 @@ impl NightshadeRuntime {
             )
             .map_err(|e| match e {
                 RuntimeError::InvalidTxError(_) => ErrorKind::InvalidTransactions,
-                RuntimeError::BalanceMismatch(e) => panic!("{}", e),
+                RuntimeError::BalanceMismatch(e) => panic!("RuntimeError::BalanceMismatch: {}", e),
```

Comment:
Don't need to indicate `RuntimeError::BalanceMismatch` in the format. Display for the error already includes it